src/formatter.js

```diff
@@ -230,6 +248,7 @@ let _formatters = {
 	'F|file' : _formatFile,
 	'K|map|MAP' : _formatMapMessage,
 	'L|line' : _formatLineNumber,
+	'column': _formatColumn,
```

Comment:
Great! When using as replacement of console.log, this was one of the things I was missing.